src/main/java/de/kittybot/kittybot/objects/GuildSettings.java

```diff
@@ -0,0 +1,176 @@
+package de.kittybot.kittybot.objects;
+
+import de.kittybot.kittybot.utils.MessageUtils;
+import de.kittybot.kittybot.utils.Utils;
+
+public class GuildSettings{
+
+	private final String guildId;
```

Comment:
why does this object need to hold the guild id? the only usage is pruning the cache with this stored id but for that refer to my previous comment

Reply:
Yes yes me dumbo :)